### DISCUSSION TITLE
Organization

### DISCUSSION THREAD

**Discussion by Something8290** [aldostools/webMAN-MOD#683] [General]
I have few a questions regarding organization in the ps3 home screen.

1. Can I mix up the pkg. installed games with .iso games into the same folder? For example, putting Metal Gear Solid 4 (.pkg installed) and Metal Gear Solid (.bin etc.) into one folder.

2. Can i hide extra game icons. For example, Resident Evil  2 has two discs and each disc has 2 tracks so the game shows up on the home menu 4 times.

3. Can i modify the icons of the default ps3 made  game folders. If not, is there a way to change the webman ps1 ps2 ps3 folder icons to their original folder icons?

**Comment by aldostools** [comment]
1-ISO games and PKG files are scanned in different folders.
Your options are:
a- Use PS3ISO for games and PKG or Packages folders on NTFS/exFAT for .pkg files. (Default behavior)
b- Rebuild the ISO image of your games. Put the .pkg files in the root of the image or in PS3_EXTRA.
If you put the .pkg files in PS3_EXTRA, you need to activate this attribute in PARAM.SFO (this is done by PS3_EXTRA creator tool)
![image](https://user-images.githubusercontent.com/974897/164705834-61f37de9-ce34-4d9a-a684-0d1275bc35f5.png)
If you put the .pkg files in the root of the ISO image, the game will not appear as a disc, but the packages can be installes with the Package Manager.
c- You can create a separate ISO image (or GAMES folder) containing only the package files.
If you create the ISO images as data disc, you can put them in BDISO or DVDISO.
If you create the ISO as game disc, you need to create the proper folder structure and put the ISO files in PS3ISO.

2- You need to modify the PS3_EXTRA flag in the ATTRIBUTE field of PARAM.SFO
If you turn it off, the PS3_EXTRA discs will not appear. Also you cannot access the content of these folders.

3- You can modify the icons of webMAN folders replacing the icons in /dev_hdd0/tmp/wm_icons
If you put a .png or .jpg with the same name of the ISO image, the game will be listed with that icon.
If you have the covers pack installed, the cover will be searched by game id + .jpg or .png

**Reply by Something8290** [reply]
Hi, first of all thank you for responding so quickly and appreciate all the work you've done. I sent a small donation for your work.
I am pretty new at all this so im struggling with your answer "1.a"  i got ps3iso games to work on NTFS but PKG or Packages folder with .pkg files don't seem to be read or show up on the menu.

And for your answer "1.C"  is there a tool to create specific iso like datadisc or game disc as you mention?

**Comment by aldostools** [comment]
First of all, thank you for  your donation. It is very helpful for the continuity of this project.

To read the PKG folder on NTFS or exFAT, it is required to run prepISO. The tool is available in the releases page.

To create data disc is very easy. You can use ImgBurn, PowerISO or any ISO tool that you prefer.

To build PS3 games is more complex, because they require special tools with a specific folder structure. I suggest that go with data discs stored in /DVDISO or /BDISO. Mount the ISO and use Package Manager to install the packages.

To learn about the folder structure, check the wiki page:
https://www.psdevwiki.com/ps3/PARAM.SFO#CATEGORY

To build the PS3ISO you can use ps3gen or Cobra_GenPS3iso. You can find them here:
https://github.com/aldostools/webMAN-MOD/tree/master/_Projects_/tools

**Comment by Something8290** [comment]
Alright, I got it to work. Thank you very much.
